client/js/domhelpers.js

```diff
@@ -31,6 +31,10 @@ export function domByTemplate(id, type) {
   return div;
 }
 
+export function mapAssetURLs(str) {
+  return str.replaceAll(/url\((["']?)\//g, 'url($1');
```

Comment:
There is valid css that will fail to match this RegExp. We should handle whitespace between the open parentheses and the string. We probably should also handle `src()` function instead of `url()` function. `src()` function can contain css `var()` function, but I think we should give up at that point.

Reply:
We could also match something similar to `/[^a-z0-9_-]\/(assets|i)\//`.

Reply:
I changed it to `str.replaceAll(/(["' (])\/(assets|i)\//g, '$1$2/')`. I think it's _very_ unlikely that this removes slashes where it shouldn't and I think it should cover all valid URL usages, right?

Reply:
Yeah, I think that is good enough.